### ISSUE TITLE
refactor doc links to trailing slashes?

### ISSUE THREAD

**Issue by felixroos** [tidalcycles/strudel#841]
currently, all the doc links point to urls without trailing slash. the reason is this now closed issue: https://github.com/tidalcycles/strudel/issues/569 with that fixed, urls with trailing slash now work.

the algolia search results actually point to the versions with trailing slash. 
when clicking on a result, the sidebar anchor links will reload the whole page because they point to the slashless version (and sometimes they might not work, not sure when exactly).

also, the algolia crawler seems to first hit the slashless urls and then the redirect, which is why the crawler ignores many urls, polluting the output.

the slashless urls only work because, for directories, github pages redirects from URLs without trailing slash to the same URL with trailing slash as mentioned here: https://github.com/vite-pwa/astro/issues/23#issuecomment-1773392276 (although the service worker intercepts these redirects and serves the cached response somehow) + astro builds each page into its own directory by default (https://docs.astro.build/de/reference/configuration-reference/#buildformat defaults to 'directory')

TLDR; just refactor the links to get a cleaner algolia crawl + less flashing when clicking around the docs

**Comment by felixroos** [comment]
I'll look into it

**Comment by felixroos** [comment]
fixed in https://github.com/tidalcycles/strudel/pull/842
